tcf_website/templatetags/custom_tags.py

```
@@ -15,3 +16,58 @@ def get_item(dictionary, key):
+    colors = [
+        "bg-primary",
+        "bg-success", 
+        "bg-info",
+        "bg-warning",
+        "bg-danger",
+        "bg-dark",
+        "bg-secondary"
+    ]
+    
+    # Create a hash of the tag name to get consistent color
+    tag_hash = int(hashlib.md5(tag_name.lower().encode()).hexdigest(), 16)
```

Ruff (0.14.1)

Probable use of insecure hash functions in `hashlib`: `md5`

(S324)